scripts/update-flake.sh

```diff
@@ -13,4 +27,14 @@ echo "Calculating SRI hash..."
 HASH=$(go run tailscale.com/cmd/nardump --sri "$OUT/pkg/mod/cache/download")
 sudo rm -rf "$OUT"
 
+echo "Updating go.mod vendorHash"
 sed -i "s#\(vendorHash = \"\)[^\"]*#\1${HASH}#" ./flake.nix
+
+# Update protoc-gen-go sha256
+echo "Updating protoc-gen-go sha256..."
+PROTOC_GEN_GO_REV=$(grep -A 20 'pkgs.buildGoModule rec' flake.nix | grep -A 10 'repo = "protobuf-go"' | grep 'rev = "v' | sed 's/.*rev = "\(.*\)".*/\1/')
```

Comment:
This is flimsy (and possible not portable across Linux/Mac/Windows); is there a more robust way we can get this?